packages/plugin-default-event-tracking-advanced-browser/src/default-event-tracking-advanced-plugin.ts

```diff
@@ -22,6 +22,7 @@ export const DEFAULT_CSS_SELECTOR_ALLOWLIST = [
   'textarea',
   'label',
   '.amp-default-track',
+  '[data-amp-default-track]',
```

Comment:
Nit: Maybe attribute should go first (preferred over class)?
```suggestion
  '[data-amp-default-track]',
  '.amp-default-track',
```